chapters/sampling-randomization-power.tex

```diff
@@ -1,529 +1,782 @@
+how that data is merged to any other data.
+
+
+%%%%% Regression discontinuity
+
+\textbf{Regression discontinuity (RD)}\sidenote{
+	\url{https://dimewiki.worldbank.org/Regression\_Discontinuity}}
+\index{regression discontinuity}
+designs exploit sharp breaks or limits
+in policy designs to separate a single group of potentially eligible recipients
+into comparable groups of individuals who do and do not receive a treatment.
+Common examples are test score thresholds and income thresholds,
+where the individuals on one side of some threshold receive
+a treatment but those on the other side do not.\sidenote{
+	\url{https://blogs.worldbank.org/impactevaluations/regression-discontinuity-porn}}
+The intuition is that, on average,
+individuals immediately on one side of the threshold
+are statistically indistinguishable from the individuals on the other side,
+and the only difference is receiving the treatment.
+In your data you need an unambiguous way
+to define which observations were above or below the cutoff.
+The cutoff determinant, or running variable,
+is often a continuous variable 
+that used to divide the sample into two or more groups. 
+Both the running variable and a categorical cutoff variable,
+should be saved in your master dataset.
+
+%%%%% IV regression
+
+\textbf{Instrumental variables (IV)}\sidenote{
+	\url{https://dimewiki.worldbank.org/Instrumental\_Variables}}
+\index{instrumental variables}
+designs, unlike the previous approaches,
+assume that the treatment effect is not directly identifiable.
+Similar to RD designs,
+IV designs focus on a subset of the variation in treatment take-up.
+Where RD designs use a \textit{sharp} or binary cutoff,
+IV designs are \textit{fuzzy}, meaning that the input does not completely determine
+the treatment status, but instead influence the \textit{probability of treatment}.
+You will need variables in your data
+that can be used to estimate the probability of treatment for each unit.
+These variables are called \textbf{instruments}.
+In IV designs, instead of the ordinary regression estimator,
+a special version called two-stage least squares (2SLS) is used
+to estimate the impact of the treatment.
+Stata has a built-in command called \texttt{ivregress},
+and another popular implementation is the user-written command \texttt{ivreg2}.
+
+
+%%%%% Matching
+
+\textbf{Matching}\sidenote{
+	\url{https://dimewiki.worldbank.org/Matching}}
+methods use observable characteristics to construct
+sets of treatment and control units
+where the observations in each set 
+are as similar as possible. \index{matching}
+These sets can either consist of exactly one treatment and one control observation (one-to-one),
+a set of observations where
+both groups have more than one observation represented (many-to-many),
+or where only one group has more than one observation included (one-to-many).
+By now you can probably guess that 
+the result of the matching needs to be saved in the master dataset.
```

Comment:
often being both the score and the resulting pairing

Reply:
I think the score is similar to instrument variables. Not certain they mean something time-invariant. And in the case of matching, you never merge the score to your analysis data, only the pairs, or the treatment assignment based on the pair.

This section will be re-written tho. So I will resolve this conversation.